frontend/server/k8s-helper.ts

```diff
@@ -84,7 +85,7 @@ if (isInCluster) {
   k8sV1CustomObjectClient = kc.makeApiClient(Custom_objectsApi);
 }
 
-function getNameOfViewerResource(logdir: string): string {
+function getNameOfViewerResource(logdir: string, tfversion: string): string {
```

Comment:
Some comments on what the viewer resource id is determined? E.g., the id depends on logdir and/or tfversion?

Reply:
The id depends only on logdir for now. I'll remove the unused parameters.